### ISSUE TITLE
eventsubmission: 403 error

### ISSUE THREAD

**Issue by JKoelman** [jemproject/JEM-Project#177]
Just wondering,.

It's possible to add a link to eventsubmission, but when someone is not allowed he will get an ugly error.

![clipboard12](https://f.cloud.github.com/assets/2514855/594378/1a9da8d0-ca7f-11e2-9295-351d9515fa9f.jpg)

And that's done by this code
![clipboard11](https://f.cloud.github.com/assets/2514855/594377/0fe12b92-ca7f-11e2-873b-1917b0e4c110.jpg)

and by this setting
![clipboard13](https://f.cloud.github.com/assets/2514855/594382/b27ab5d0-ca7f-11e2-8605-8f1631ae5807.jpg)

Should we change the error notice or just leave it as it is?.
Basically you should not the link when you don't want to allow registered users to post.
